test/libgit2.jl

```diff
@@ -1014,8 +1014,11 @@ mktempdir() do dir
 
             # Find a hostname that maps to the loopback address
             hostnames = ["localhost"]
+
+            # In minimal environments a hostname might not be available (issue #20758)
             try
-                # In minimal environments a hostname might not be available (issue #20758)
+                # Some environments, namely Macs, the hostname "macbook.local" bound to the
```

Comment:
I think this is clear enough. Comments in internal files don't need to be 100% grammatically perfect.